database/migrations/platform/2023_09_15_000000_update_player_games_player_achievement_sets.php

```diff
@@ -0,0 +1,82 @@
+            $table->string('update_status')->nullable()->after('achievement_set_version_hash');
+
+            $table->unsignedDecimal('completion_percentage', 10, 9)->nullable()
+                ->change();
+
+            $table->unsignedInteger('achievements_beat')->nullable()->after('achievements_unlocked_hardcore');
+            $table->unsignedInteger('achievements_beat_unlocked')->nullable()->after('achievements_beat');
+            $table->unsignedInteger('achievements_beat_unlocked_hardcore')->nullable()->after('achievements_beat_unlocked');
+
+            $table->unsignedDecimal('beaten_percentage', 10, 9)->nullable()->after('achievements_beat_unlocked_hardcore');
+            $table->unsignedDecimal('beaten_percentage_hardcore', 10, 9)->nullable()->after('beaten_percentage');
+
+            $table->jsonb('beaten_dates')->nullable()->after('time_taken_hardcore');
+            $table->jsonb('beaten_dates_hardcore')->nullable()->after('beaten_dates');
+
+            $table->timestampTz('beaten_at')->nullable()->after('completion_dates_hardcore');
+            $table->timestampTz('beaten_hardcore_at')->nullable()->after('beaten_at');
+
+            $table->unsignedInteger('points_hardcore')->nullable()->after('points');
+        });
+
+        Schema::table('player_achievement_sets', function (Blueprint $table) {
+            $table->unsignedDecimal('completion_percentage', 10, 9)->nullable()
+                ->change();
+
+            $table->unsignedInteger('achievements_beat')->nullable()->after('achievements_unlocked_hardcore');
+            $table->unsignedInteger('achievements_beat_unlocked')->nullable()->after('achievements_beat');
+            $table->unsignedInteger('achievements_beat_unlocked_hardcore')->nullable()->after('achievements_beat_unlocked');
+
+            $table->unsignedDecimal('beaten_percentage', 10, 9)->nullable()->after('achievements_beat_unlocked_hardcore');
+            $table->unsignedDecimal('beaten_percentage_hardcore', 10, 9)->nullable()->after('beaten_percentage');
+
+            $table->jsonb('beaten_dates')->nullable()->after('time_taken_hardcore');
+            $table->jsonb('beaten_dates_hardcore')->nullable()->after('beaten_dates');
+
+            $table->timestampTz('beaten_at')->nullable()->after('completion_dates_hardcore');
+            $table->timestampTz('beaten_hardcore_at')->nullable()->after('beaten_at');
+
+            $table->unsignedInteger('points_hardcore')->nullable()->after('points');
+        });
+    }
+
+    public function down(): void
+    {
+        Schema::table('player_games', function (Blueprint $table) {
+            $table->dropColumn('achievements_beat');
+            $table->dropColumn('beaten_percentage');
+            $table->dropColumn('beaten_percentage_hardcore');
+            $table->dropColumn('beaten_dates');
+            $table->dropColumn('beaten_dates_hardcore');
+            $table->dropColumn('beaten_at');
+            $table->dropColumn('beaten_hardcore_at');
```

Comment:
```suggestion
            $table->dropColumn('achievement_set_version_hash');
            $table->dropColumn('achievements_beat');
            $table->dropColumn('beaten_percentage');
            $table->dropColumn('beaten_percentage_hardcore');
            $table->dropColumn('beaten_dates');
            $table->dropColumn('beaten_dates_hardcore');
            $table->dropColumn('beaten_at');
            $table->dropColumn('beaten_hardcore_at');
            $table->dropColumn('update_status');
```

Added achievement_set_version_hash and update_status

Reply:
Thanks! Seems i missed a few more columns: https://github.com/RetroAchievements/RAWeb/pull/1874/commits/f340b836f4467a6b838900025e4851647bd7ebef